docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2876,6 +2876,34 @@ Enables TLS client authentication for the JSON-RPC HTTP service. The default is
     You must specify [`--rpc-http-tls-ca-clients-enabled`](#rpc-http-tls-ca-clients-enabled) and/or
     [`rpc-http-tls-known-clients-file`](#rpc-http-tls-known-clients-file).
 
+### `rpc-http-tls-cipher-suite`, `rpc-http-tls-cipher-suites`
+
+=== "Syntax"
+
+    ```bash
+    --rpc-http-tls-cipher-suite=<cipherSuiteName>[, <cipherSuiteName>...]
+    ```
+
+=== "Example"
+
+    ```bash
+    --rpc-http-tls-cipher-suite=TLS_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_RPC_HTTP_TLS_CIPHER_SUITE=TLS_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    rpc-http-tls-cipher-suite=["TLS_AES_256_GCM_SHA384","TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384","TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"]
+    ```
+
+A list of comma separated TLS cipher suites to support.
```

Comment:
```suggestion
A list of comma-separated TLS cipher suites to support.
```

Maybe note here whether the singular and plural aliases can be used interchangeably, or a plural list requires the plural alias.